### PR TITLE
Mobilito icons 968

### PR DESCRIPTION
**Pull request by JeffAbrahamson**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,9 +8,9 @@ venv.vagrant
 Pipfile*
 
 # Django
-db.sqlite3
-media/
-static/
+/transport_nantes/db.sqlite3*
+/transport_nantes/media/
+/transport_nantes/static/
```
Comment (Shriukan33) [inline .gitignore:13]: We don't have these paths for media and static, they are transport_nantes/<app_name>/static

**Review by Shriukan33 — CHANGES_REQUESTED** [review]
LGTM for images, the gitignore thing however I think I do understand the intention, I don't think it does that. 

You tell git to ignore transport_nantes directory /media and /static subdirs, but we don't have those anyway ? Also we don't want to ignore the app's static subdirs, as it makes JS and CSS ignored.

File: .gitignore (hunk 1 above)
Comment (JeffAbrahamson) [reply .gitignore:13]: Exactly, and we very much want content in `transport_nantes/*/static` to  not be ignored.

What we want ignored is the stuff that collectstatic accumulates in `transport_nantes/static/` or that we test upload to `transport_nantes/media`.

Comment (Shriukan33) [reply .gitignore:13]: Hmm, I see, my statics don't go in those but maybe that's my local settings. LGTM then !

**Review by Shriukan33 — APPROVED** [review]

**Comment by JeffAbrahamson** [comment]
Where do you put your static and media files?  Those are the usual places to configure for dev.

**Comment by JeffAbrahamson** [comment]
We should probably do something to make this more likely to be uniform among devs.

**Comment by Shriukan33** [comment]
> Where do you put your static and media files? Those are the usual places to configure for dev.

That's maybe because I don't run collectstatic locally, but everything stays in app subdirs

**Comment by JeffAbrahamson** [comment]
> > Where do you put your static and media files? Those are the usual places to configure for dev.
> 
> That's maybe because I don't run collectstatic locally, but everything stays in app subdirs

That seems valid, but then it's also unaffected by this change!

**Pull request merged**